master_pose:
 - 0.0273209
 - -0.1016368
 - -0.6928710
